mlkem/poly.c

```diff
@@ -272,6 +272,7 @@ void poly_decompress(poly *r, const uint8_t a[KYBER_POLYCOMPRESSEDBYTES])
 *              - r: pointer to output byte array
 *                   (of KYBER_POLYBYTES bytes)
 **************************************************/
+#if !defined(MLKEM_USE_AARCH64_ASM)
```

Comment:
Move the pre-procesor conditional inside of `poly_tobytes`

Reply:
I actually think it's cleaner this way. Also, for functions like the basemul ASM which need extra static variables, you end up with multiple if-def blocks if you mandate one within the function-body itself.

I'm happy to use the above style (single if-def around function def) everywhere if you're happy with it.

Reply:
I've done that for now, have a look.

Reply:
Yes, I'm fine with that.